scripts/generateSSL.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+cd ./app
+
+if [ ! -d "keys" ]; then
+  mkdir keys
+fi
+
+cd ./keys
+
+rm *domain*
+
+echo Generating SSL Key/Cert pair...
+
+openssl req \
+  -newkey rsa:2048 -nodes -keyout domain.key \
```

Comment:
Should be completely automatic and not prompt for any entry, see https://unix.stackexchange.com/questions/104171/create-ssl-certificate-non-interactively

Reply:
Tested, and implemented a script from this post. Works as intended, completely automatic!